module/behaviour/strategy/PS3Walk/src/PS3Walk.cpp

```diff
@@ -51,14 +51,14 @@ namespace module::behaviour::strategy {
                         case AXIS_LEFT_JOYSTICK_HORIZONTAL:
                             // y is left relative to robot
                             // strafe[1] = -event.value;
-                            rotationalSpeed = float(-event.value);
+                            rotationalSpeed = static_cast<float>(-event.value);
```

Comment:
What rule is forcing you to change to static casts here?
I don't think it should be required

Reply:
It's not required. I just like static casts

Reply:
Fair enough :P I like functional casts but I am aware it's a stylistic choice so idm